### ISSUE TITLE
LEGITIMATE SITE BLOCKED

### ISSUE THREAD

**Issue by CGA-251** [MetaMask/eth-phishing-detect#8106]
Hello,

We recently purchased the domain:
Crypto-giveaway.com 

It has come to our attention that due to previous ownership the domain has been placed on the MetaMask domain warning list. 

We’ve made great efforts to develop the site placing a particular emphasis on helping users stay secure and not fall victim to any scam or fraudulent effort. 

We kindly request that you remove us from your warning list. We’re fellow believers in decentralization and would like a chance to prove our honest reputation. If you would like any further information please let us know. 

Thank you in advance!

**Comment by CGA-251** [comment]
Hello - Bumping this for visibility. 
Please let us know anything we can do to assist. 
Thank you!

**Comment by CGA-251** [comment]
@devenblake We’d really appreciate if you could take a look at the above. 
This is a legitimate website. If you need any information please let us know. 
Thank you in advance!

**Comment by CGA-251** [comment]
Hello - continuing to bump this for visibility.

Thank you

**Comment by CGA-251** [comment]
Kindly asking once more, to please review and remove our site.

Thank you,

**Comment by AlexHerman1** [comment]
@CGA-251  can you provide more details about your project?

**Comment by CGA-251** [comment]
Thank you for your response!

Essentially, our project is focused on 2 things:
- Providing competitions & prizes which will (hopefully) generate interest and help develop our "brand"
- Education (via our blog and newsletter)

We understand number 1 is met with scorn and cynicism but we just want a chance to prove ourselves. We bought the domain not knowing that it was blacklist well in the past. We've also gone to great lengths to help our audience protect themselves from common scams etc (please see our security guide page) and to make it clear that we will never ask them for anything. 

Please take a look at our site and if you have any feedback or concerns please let us know. 

Thank you once again

**Comment by devenblake** [comment]
I spent a little bit of time looking at the site. I didn't see any lora ipsa or other templates strewn about and the content on each page I found seemed acceptably well thought out. I particularly like the [Security page](https://crypto-giveaway.com/security/) which goes over how to avoid common scams.

[Here's the site WHOIS lookup](https://www.whois.com/whois/crypto-giveaway.com). Registered 2021, updated 2022-07-29. The site was blocked in [commit 2b0172e29](https://github.com/MetaMask/eth-phishing-detect/commit/2b0172e29) in 2019 (before the current registration), [urlscanned](https://urlscan.io/result/2fb329bd-f7e3-4ab1-b856-70265fa208c8/) [twice](https://urlscan.io/result/2cb389a2-1dde-417c-8c51-b66d9eac4cdb/) and impersonating Coinbase both times.

There's good reason to believe this is site is no longer in nefarious hands. Possibly worth noting though; we have many similarly named sites denylisted as well:
```
2b0172e29 2019-11-25 16:15:04 +0000  6783) "crypto-giveaway.com",
adca3ea73 2018-11-23 19:17:08 +0000  9871) "crypto-giveaways.com",
306735a09 2018-11-20 00:23:09 +0000  9984) "crypto-giveaway.org",
424b9aa20 2018-09-06 17:07:23 +0100 11599) "crypto-giveaway.xyz",
87be619fa 2018-09-03 00:18:56 +0100 11678) "crypto-giveaway.net",
1f30ca2a7 2018-08-14 15:14:48 +0100 12165) "crypto-giveaway.top",
```

**Comment by CGA-251** [comment]
Thank you, we appreciate your feedback. And to confirm, we have no affiliation with any of the other domains listed above.

**Comment by CGA-251** [comment]
Hello following up here. Can we action this?

Thank you!

**Comment by CGA-251** [comment]
Bumping this again, can you please push this through?

Thank you!

**Comment by AlexHerman1** [comment]
fixed via #8216
